projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { By } from '@angular/platform-browser';
+import { OpenInAppComponent } from './open-in-app.component';
+
+describe('OpenInAppComponent', () => {
+  let fixture: ComponentFixture<OpenInAppComponent>;
+  let component: OpenInAppComponent;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [OpenInAppComponent],
+      providers: [{ provide: MAT_DIALOG_DATA, useValue: { redirectUrl: 'mockRedirectUrl' } }]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(OpenInAppComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should redirect to app when click on `Open in App` button` ', async () => {
+    let currentLocation: any;
```

Comment:
Can we specify the type instead of using `any`?

Reply:
specified type instead of any.